cms/static/js_test_squire.yml

```diff
@@ -75,6 +75,7 @@ src_paths:
     - common/js
     - edx-pattern-library/js
     - edx-ui-toolkit/js
+    - js/vendor/afontgarde
```

Comment:
This isn't even the correct path, is it? If it's not needed here, just don't add it.